src/models/stores.ts

```diff
@@ -1,3 +1,4 @@
+import uuid from "uuid";
```

Comment:
This import appears not to be used. Was the plan to use the uuid instead of "0" in defining the user below? As is, the changes don't seem to have any effect.